pkg/preflights/host-preflight.yaml

```diff
@@ -355,4 +363,24 @@ spec:
               message: "Neither 'nameserver localhost' nor 'nameserver 127.0.0.1' can be present in resolv.conf. Remove them to continue."
           - pass:
               when: "false"
-              message: "Neither 'nameserver localhost' nor 'nameserver 127.0.01' is present in resolv.conf"
\ No newline at end of file
+              message: "Neither 'nameserver localhost' nor 'nameserver 127.0.01' is present in resolv.conf"
+    - textAnalyze:
+        checkName: 127.0.0.1 in resolv.conf
+        fileName: host-collectors/run-host/resolv.conf.txt
+        regex: 'nameserver 127.0.0.1'
+        outcomes:
+          - fail:
+              when: "true"
+              message: "resolv.conf should not reference '127.0.0.1', this will prevent coredns within the cluster from resolving"
+          - pass:
+              when: "false"
+              message: "resolv.conf does not reference '127.0.0.1'"
```

Comment:
The two resolv.conf preflights were merged into one, which it looks like is above, so this one can be removed.
```suggestion
```